chart/templates/redis/redis-statefulset.yaml

```
@@ -22,6 +22,7 @@
 {{- $nodeSelector := or .Values.redis.nodeSelector .Values.nodeSelector }}
 {{- $affinity := or .Values.redis.affinity .Values.affinity }}
 {{- $tolerations := or .Values.redis.tolerations .Values.tolerations }}
+{{- $securityContext := include "airflowSecurityContext" (list . .Values.redis) }}
```

```suggestion
{{- $securityContext := include "localSecurityContext" .Values.redis }}
```

This should use `localSecurityContext` instead. We don't want the global Airflow security context to apply to Redis.